Makefile

```diff
@@ -150,6 +150,16 @@ $(BUILD_DIR)/%.bin: $(BUILD_DIR)/%.elf | $(BUILD_DIR)
 $(BUILD_DIR):
 	mkdir -p $@
 
+
+debug: build/ODriveFirmware.elf
+	{ openocd -c gdb_port 3333 -c telnet_port 4444 -f interface/stlink-v2.cfg -f target/stm32f4x_stlink.cfg -c 'echo "Started by GNU ARM Eclipse"' & } && echo $$!>.openocd.pid
+	arm-none-eabi-gdb -ix .gdbinit $<; cat .openocd.pid | xargs kill -2; rm .openocd.pid
```

Comment:
Actually, this way of launching two processes and getting the pid seems overcomplicated. At work we use this command:
`arm-none-eabi-gdb ${NAME}.elf -x openocd.gdbinit`
where openocd.gdbinit contains:
```
target remote | openocd -f "interface/stlink-v2.cfg" -f "target/stm32f4x_stlink.cfg" -c "gdb_port pipe; log_output openocd.log"
monitor reset halt
```

See if you can get something like that to work?